community/modules/compute/gke-node-pool/README.md

```diff
@@ -142,15 +142,15 @@ limitations under the License.
 | Name | Version |
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0 |
-| <a name="requirement_google"></a> [google](#requirement\_google) | >= 4.60.0, < 5.0 |
-| <a name="requirement_google-beta"></a> [google-beta](#requirement\_google-beta) | >= 4.60.0, < 5.0 |
+| <a name="requirement_google"></a> [google](#requirement\_google) | ~> 4.69.1 |
```

Comment:
I would talk to Nick here, I assume he left a large range like this by design. Also, you are not actually upgrading it here, if anything you are constraining it...

Reply:
I assumed we should update this since we are [pinning to minor versions](https://cloud.google.com/docs/terraform/best-practices-for-terraform#minor-provider-versions) everywhere else.  But I agree, getting Nick's input is probably a good idea here.